client/packages/requisitions/src/RnRForms/ListView/ProgramSearchInput.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from 'react';
 import { Autocomplete } from '@openmsupply-client/common';
-import { ProgramFragment } from '../../api/operations.generated';
-import { useProgramList } from '../../api';
+import { ProgramFragment } from '../api/operations.generated';
+import { useProgramList } from '../api';
 
 type ProgramSearchInputProps = {
```

Comment:
Bit of a nit, seeing as this is only used by the R&R forms, but I wonder if this should live in programs still? Or system?

We have the programs list query defined in both the program and r&r operations files now, which I think is out of line with existing patterns?

Reply:
true, will consolidate 🙏